sites/kit.svelte.dev/vite.config.js

```diff
@@ -2,13 +2,29 @@ import { sveltekit } from '@sveltejs/kit/vite';
 import * as path from 'path';
 import { imagetools } from 'vite-imagetools';
 
+const supportedExtensions = ['.png', '.jpg', '.jpeg'];
+
 /** @type {import('vite').UserConfig} */
 const config = {
 	assetsInclude: ['**/*.vtt'],
 
 	logLevel: 'info',
 
-	plugins: [imagetools(), sveltekit()],
+	plugins: [
+		imagetools({
+			defaultDirectives: (url) => {
+				const extension = path.extname(url.pathname);
+				if (supportedExtensions.includes(extension)) {
+					return new URLSearchParams({
+						format: 'avif;webp;' + extension,
```

Comment:
ah crap, i guess this should be this now? though it seems to work anyway...

```suggestion
						format: 'avif;webp;' + extension.slice(1),
```

Reply:
gonna leave it as-is, if it ain't broke etc